zerver/lib/push_notifications.py

```diff
@@ -1680,15 +1679,11 @@ def handle_push_notification(user_profile_id: int, missed_message: dict[str, Any
     )
     logger.info("Sending push notifications to mobile clients for user %s", user_profile_id)
 
-    # TODO: We plan to offer a personal, realm-level, and server-level setting
-    # to require all notifications to be end-to-end encrypted. When either setting
-    # is enabled, we skip calling 'send_push_notifications_legacy'.
```

Comment:
Removing this setting related comment as #35436 will be merged soon, so not a lot of importance to manage this comment.